examples/classification.ipynb

```diff
@@ -18,7 +18,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "model = OpenAI(api_key=\"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\")\n",
```

Comment:
Open api key published

Reply:
Thanks for the headsup, I also noticed a bit late but revoked it :D